### ISSUE TITLE
Missing bypass blocks

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1202]
The correct [bypass blocks required by Success Criteria 2.4.1 in WCAG 2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html) is reportedly missing.

I'm able to to navigate using the tab key so I guess the issue lays in the implementation.

**Comment by StianJul** [comment]
@torgeirl, do you have any examples?

**Comment by torgeirl** [comment]
> @torgeirl, do you have any examples?

@StianJul: I'll ask!

**Comment by torgeirl** [comment]
> > @torgeirl, do you have any examples?
> 
> @StianJul: I'll ask!

In general, page menus are a good use case for this. The front page menus are lighter in Devilry so that doesn't come up as often, but in Devilry there are pages where the number of breadcrumbs elements get high.

One example is the feedback feed, with usually five objects:
1. «Devilry» (front page)
2. «Student» (student dashboard)
3. «Your courses»
4. `<coursename.semester>`
5. `<assignmentshortname>`

An example of such a bypass block, taking from uio.no's frontpage:
```
<nav id="hidnav-wrapper" aria-label="Jump to content">
  <ul id="hidnav">
    <li><a href="#total-main">Jump to main content</a></li>
  </ul>
</nav>
```
